flutter-studio/src/io/flutter/profiler/FlutterAllMemoryData.java

```diff
@@ -47,9 +49,13 @@ public ThreadSafeData() { }
         if (data.x >= xRange.getMin()) {
           // TODO(terry): Hack for now to get last item so we don't flicker. Consider using a for/loop instead of iterator.
           if (lastData != null && outData.isEmpty()) {
-            outData.add(lastData);
+            // TODO(terry): Need to copy data because of modification done in LineChart for stacked plotting.
```

Comment:
Looks like you are copying out the data so perhaps remove the TODO(terry): as that made it look like we weren't copying.
Having the comment is great to explain why the data is copied.

Reply:
Right, its an old comment before I copied the data to workaround a problem with the adt-ui code.  I'll remove the comment until I no longer have to copy the data.